### PR TITLE
fix: passive healthy code review.

### PR DESCRIPTION
**Pull request by 840963657**
### Description

Fixes # (issue)

### Checklist

- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Comment by 840963657** [comment]
when resp_status == status, there is no need to perform other loop operations.

**Review by tzssangglass — APPROVED** [review]

File: apisix/init.lua [hunk 1]
```diff
@@ -724,6 +724,7 @@ local function healthcheck_passive(api_ctx)
                                            port or api_ctx.balancer_port,
                                            host,
                                            resp_status)
+                return
```
Comment (spacewander) [inline apisix/init.lua:727]: We should use break here?

Comment (840963657) [reply apisix/init.lua:727]: In the second loop, I think break and return have the same effect

Comment (tzssangglass) [reply apisix/init.lua:727]: we should use break here.

Comment (spacewander) [reply apisix/init.lua:727]: Sorry, I can't get your point. What does "the second loop' mean?

Comment (840963657) [reply apisix/init.lua:727]: Loop of the 727 line, the function ends when the loop terminates, so think break and return have the same effect
image

Comment (tzssangglass) [reply apisix/init.lua:730]: in L730, we should use `break`?

**Pull request closed without merging**